components/NavigationBar/src/MDCNavigationBar.m

```diff
@@ -146,8 +146,10 @@ - (void)commonMDCNavigationBarInit {
   _titleLabel.textAlignment = NSTextAlignmentCenter;
   _leadingButtonBar = [[MDCButtonBar alloc] init];
   _leadingButtonBar.layoutPosition = MDCButtonBarLayoutPositionLeading;
+  _leadingButtonBar.delegate = self;
```

Comment:
Can you add a unit test that the NavBar's buttonBar frame will grow/shrink if the title of the item changes?

Reply:
Done.